src/gui/log/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+target_sources(qbt_gui PRIVATE
```

Comment:
why not put them in gui/cmakelists.txt?

Reply:
It's inconsistent. Some subfolder sources are included directly but the others via "subprojects" (in qmake project too). I have no strong preference about what way is better but I want to make it consistent (at least inside the same project).

Reply:
IMO a corresponding log.pri should be added too. I don't insist but hope you don't mind adding it.

Reply:
>IMO a corresponding log.pri should be added too. 

There are some other subfolders that have no . PRI file.
> I don't insist but hope you don't mind adding it.

I don't mind. But I would prefer to do it in the next "clean up" PR. I don't want to linger too long on such PRs, so as not to be distracted from more important matters. If there are no complaints about its contents, I would not like to stay here any longer.

Reply:
>IMO a corresponding log.pri should be added too. 

Or maybe it's better to go in a different direction and get rid of these .pri files in subdirectories (and of CMakeLists.txt as well)? Just include all in `gui.pri` and `gui/CMakeLists.txt` directly?

Reply:
>Just include all in gui.pri and gui/CMakeLists.txt directly?

👍